daemon/src/taker_cfd.rs

```diff
@@ -119,15 +119,20 @@ impl<O, W> Actor<O, W> {
         self.current_maker_offers = takers_perspective_of_maker_offers.clone();
         tracing::trace!("new maker offers {:?}", takers_perspective_of_maker_offers);
 
-        self.projection_actor
+        if let Err(e) = self
+            .projection_actor
             .send(projection::Update(
                 takers_perspective_of_maker_offers.clone(),
             ))
-            .await?;
-
-        Ok(())
+            .await
+        {
+            tracing::warn!("Failed to send current offers to projection actor: {e:#}");
```

Comment:
I guess it makes sense to change this to not report back to the caller - the protocol actor won't know what to do with this problem.